packages/features/tasker/tasks/triggerNoShow/triggerGuestNoShow.ts

```diff
@@ -1,12 +1,29 @@
+import { prisma } from "@calcom/prisma";
 import { WebhookTriggerEvents } from "@calcom/prisma/enums";
 
-import { calculateMaxStartTime, sendWebhookPayload, prepareNoShowTrigger } from "./common";
+import { calculateMaxStartTime, sendWebhookPayload, prepareNoShowTrigger, log } from "./common";
+
+const markAllGuestNoshowInBooking = async ({ bookingId, hostsThatJoinedTheCall }) => {
+  try {
+    const hostsThatJoinedTheCallEmails = hostsThatJoinedTheCall.map((h) => h.email);
+
+    await prisma.attendee.updateMany({
+      where: {
+        bookingId,
+        email: { notIn: hostsThatJoinedTheCallEmails },
+      },
+      data: { noShow: true },
+    });
+  } catch (err) {
+    log.error("Error marking guests as no show in booking", error);
+  }
+};
```

Comment:
host could also be stored in Attendee table so we make sure we only mark guests as no show